README.md

```diff
@@ -50,3 +50,164 @@ FAQ
 ---
 
 Go to the [FAQ](https://github.com/Elao/symfony-standard/wiki/FAQ)
+
+Install ElaoStrap (optionnal)
+---
+
+Install our custom twitter bootstrap Symfony bundle : [ElaoStrap](https://github.com/Elao/ElaoThemeElaoStrapBundle)
+    
+Update your /package.json. Here an example :
+
+    {
+      "name": "project-name",
```

Comment:
You are using 2 spaces indents for the first level and the author object, but 4 spaces indents everywhere else. Maybe we can agree to use only 4 spaces indents?